app/uk/gov/nationalarchives/omega/editorial/connectors/ApiConnector.scala

```diff
@@ -74,6 +74,11 @@ class ApiConnector @Inject() (
       handle(SID.GetPlacesOfDeposit, Json.stringify(Json.toJson(getPlacesOfDeposit)))
         .flatMap(parse[Seq[PlaceOfDeposit]])
 
+  def getLegalStatusesSummary(getLegalStatuses: GetLegalStatuses): IO[Seq[LegalStatus]] =
```

Comment:
I'd suggest renaming it to `getLegalStatuses`.

Reply:
Renamed

Reply:
Confirmed.